app/src/app/components/contents/manageAccess/manageRolesColumns.tsx

```diff
@@ -0,0 +1,73 @@
+import { createColumnHelper } from "@tanstack/react-table";
+import { RoleWithRelationships } from "./types/RoleWithRelationships";
+import { constructPermissionName } from "../../../../lib/constructPermissionName";
+import { Button } from "../../Base/Button";
+import { EllipsisVertical, Trash2 } from "lucide-react";
+import { ScrollArea } from "../../Base/ScrollArea";
+
+const columnHelper = createColumnHelper<RoleWithRelationships>();
+
+export const manageRolesColumns = [
+  columnHelper.accessor("name", {
+    header: "Role",
+    cell: ({ getValue }) => {
+      return <div>{getValue()}</div>;
+    }
+  }),
+  columnHelper.accessor("rolePermissions", {
+    header: "Permissions",
+    cell: ({ getValue }) => {
+      const permissions = getValue();
+
+      return (
+        <ScrollArea className="h-12">
+          <div className="flex flex-wrap italic gap-0.5  text-xs pl-0.5">
+            {permissions?.length === 0
+              ? "None"
+              : permissions.map((permission, index) => (
+                  <div key={permission.id}>
+                    {index === permissions.length - 1
+                      ? constructPermissionName(permission)
+                      : constructPermissionName(permission) + ","}
+                  </div>
```

Comment:
Any particular reason to make these separate divs? could just do a map and join on the permissions and put that string into a single element.

Reply:
the only reason was to control the spacing b/w elements ... especially vertial